tests/manual-test-cases/Group5-Functional-Tests/5-3-Enhanced-Linked-Mode.robot

```diff
@@ -18,6 +18,10 @@ Resource  ../../resources/Util.robot
 Suite Setup  Nimbus Suite Setup  Enhanced Link Mode Setup
 Suite Teardown  Run Keyword And Ignore Error  Nimbus Cleanup  ${list}
 
+*** Variables ***
+${NIMBUS_LOCATION}  sc
+${NIMBUS_LOCATION_FULL}  NIMBUS_LOCATION=${NIMBUS_LOCATION}
```

Comment:
Can we rename NIMBUS_LOCATION_FULL to ENV_NIMBUS_LOCATION or NIMBUS_LOCATION_ENV?